docs/policy/service-changes-2018.md

```diff
@@ -0,0 +1,188 @@
+Service Changes 2018
+====================
+
+As you may have heard, the Open Science Grid (OSG) is undergoing some changes to its fabric of services.
+Some services will be retired, most services will be migrated with little planned effect on sites,
+and some services will be migrated with greater effect on sites.
+This document is intended to guide OSG Site Administrators through these changes.
```

Comment:
On line 7 you capitalize Site Administrators and on line 8 you do not. I vote not, for both.